templates/new-class-stats.html

```diff
@@ -0,0 +1,87 @@
+{% extends "auth.html" %}
+
+{% block regular_content %}
+<div>
+
+    <div class="flex relative">
+        <div class="mx-auto "><h1>Class: </h1></div>
+        <button class="green-btn ml-auto" id="to_class_button"
+                onclick="window.location.href='/for-teachers/class/{{class_info.id}}'">
+          {{_('back_to_class')}}
+        </button>
+    </div>
+
+    <div class="shadow-md stats mt-4 p-2">
+        <div class="flex flex-wrap justify-center">
+            {% for student in class_info.students %}
+                <div class="student-tile shadow inline-block border px-6 py-2 m-4 hover:bg-green-400 rounded-lg transition-all duration-500 ease-in-out"
+                     onclick="this.classList.add('bg-green-400'); tileClicked(event, '{{ student }}')">
+                    <p class="font-bold text-lg">{{ student }}</p>
+                    <p>Success rate: </p>
+                    <p>Success rate: </p>
+                    <p>Highest level: </p>
+                    <p>Quiz Avg: </p>
+                </div>
+            {% endfor %}
+        </div>
+        <div id="big-tile" class="hidden w-3/4 shadow-2xl border p-2 mx-auto rounded-lg"></div>
+    </div>
+
+</div>
+
+<script>
```

Comment:
As you said yourself already, some of this JavaScript's effects can probably be replaced with CSS. Please avoid using  JavaScript unless absolutely necessary. If you do add JavaScript, please don't add it in inline. JavaScript should go into the `static/js/*.ts` files (pick a file that makes sense).

Reply:
Yes, I added the javaScript file! I will slowly remove some javaScript functions, once I have found matching tailwind CSS functionality that can replace it.